human_experiments/scripts/run_minecraft_metadata_validation.py

```diff
@@ -30,16 +33,18 @@ def check_time_difference(trial_start, trial_end):
 
 def read_subject_id(TrialMessages):
     #Display subject IDs
-    sleep(0.25)
     print(colored('\n[Status] Subject info:', 'red', attrs=['bold']))
     for idx, sub in enumerate(TrialMessages[0]['data']['metadata']['trial']['subjects']):
         print(colored('\t Subect ID','magenta'), idx,":", sub)
 
     #Display trial name
-    sleep(0.25)
     print(colored('\n[Status] Mission name:', 'red', attrs=['bold']),TrialMessages[0]['data']['metadata']['trial']['name'])
 
 def read_metadata_as_json(path):
+    '''
+    This function reads .metadata file as json loads, which makes sure header is 'event' and 
+    message is 'Event:MissionState', then reads mission start and stop timestamp from json_message["msg"]["timestamp"]
```

Comment:
header is 'event' -> .header.message_type = "event"

message is 'Event:MissionState' -> .msg.sub_type = "Event:MissionState"

Reply:
json_message["msg"]["timestamp"] -> .msg.timestamp